sdk/netapp/arm-netapp/CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
 # Release History
+
+## 21.6.0 (2025-07-25)
+
+### Features Added
+  - Interface SubscriptionQuotaItemList has a new optional parameter nextLink
```

Comment:
from changelog side it seems we have one operation from non-paging to paging? could you help analyze the real change for this api version? do we have swgger breaking?